src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllProvider.cs

```diff
@@ -42,19 +42,5 @@ public virtual IEnumerable<string> GetSupportedFixAllDiagnosticIds(CodeFixProvid
         /// Gets fix all occurrences fix for the given fixAllContext.
         /// </summary>
         public abstract Task<CodeAction> GetFixAsync(FixAllContext fixAllContext);
-
-        internal virtual Task<CodeAction> GetFixAsync(
```

Comment:
Another case of evil internal virtual methods on public type. Thankfully, they were only needed from the context of a specific sub-type, so I was able to move them directly to that sub-type.

Reply:
👍